app.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "Vector",
+    "description": "Vector is a lightweight tool for building observability pipelines. (With easy setup for CloudWatch!)",
+    "repository": "https://github.com/timberio/vector",
+    "logo": "https://vector.dev/press/vector-icon.png",
+    "keywords": ["observability"],
+    "buildpacks": [
+        { "url": "https://github.com/timberio/vector-heroku-buildpack" }
+    ],
+    "env": {
+        "VECTOR_TOML_URL": {
+            "description": "The URL of the `vector.toml` you use to use. (Refreshed on restart! For static, provision a `vector.toml` with your app.)",
+            "required": false
+        },
+        "AWS_ACCESS_KEY_ID": {
+            "description": "An AWS key ID (if needed, for sinking to CloudWatch)",
+            "required": false
+        },
+        "AWS_SECRET_ACCESS_KEY": {
+            "description": "An AWS access key (if needed, for sinking to CloudWatch)",
+            "required": false
+        },
+        "AWS_REGION": {
+            "description": "An AWS region (if needed, for sinking to CloudWatch). Examples: `ca-central-1`, `us-east-1`",
+            "required": false
+        },
+        "AWS_CLOUDWATCH_STREAM_NAME": {
+            "description": "An AWS CloudWatch stream name (if needed, for sinking to CloudWatch. Vector can create this for you.)",
+            "required": false
+        },
+        "AWS_CLOUDWATCH_GROUP_NAME": {
+            "description": "An AWS CloudWatch stream group name (if needed, for sinking to CloudWatch. You need to create this yourself.)",
+            "required": false
+        },
```

Comment:
@Hoverbear are these necessary given that we no longer prefer Cloudwatch? This is really the big thing for me. I want to make sure deploying Vector as a Heroku button is not restricted to AWS or Cloudwatch.

Reply:
We can totally yank them! :) Where would you like the logs to go then?

Reply:
Couldn't we base it entirely on the remote Vector config?

Reply:
We could! Yeah! Okay! I'll refactor this :)